NanoAnalysis/test/runLocal.py

```diff
@@ -15,7 +15,8 @@
 #SampleToRun = "MCsync_Rereco"
 #SampleToRun = "MCsync_UL"
 #SampleToRun = "Data2022"
-SampleToRun = "MC2022"
+# SampleToRun = "MC2022"
+SampleToRun = "MELA_Test"
```

Comment:
Adding a set for these tests is a good idea, but please do not change the default.